composites/Plugin/SnippetEditor/components/SnippetEditorFields.js

```diff
@@ -156,19 +180,37 @@ class SnippetEditorFields extends React.Component {
 
 		this.uniqueId = uniqueId( "snippet-editor-field-" );
 
+		this.state = {
+			isSmallerThanMobileWidth: false,
+		};
+
 		this.setRef = this.setRef.bind( this );
+		this.setEditorRef = this.setEditorRef.bind( this );
+		this.triggerReplacementVariableSuggestions = this.triggerReplacementVariableSuggestions.bind( this );
+		this.debouncedUpdateIsSmallerThanMobileWidth = debounce( this.updateIsSmallerThanMobileWidth.bind( this ), 500 );
```

Comment:
Maybe a shorter debounce value could work, for example WordPress uses 200 for its `wp-window-resized` custom event. Any specific reason for 500?

Reply:
No specific reason, will change to 200.